generators/mail/templates/common/manifest.xml

```diff
@@ -7,6 +7,7 @@
   <DefaultLocale>en-US</DefaultLocale>
   <DisplayName DefaultValue="<%= projectDisplayName %>" />
   <Description DefaultValue="[Outlook Add-in description]"/>
+  <SupportUrl DefaultValue="http://contoso.com/support" />
```

Comment:
The reason I added this entry is that the manifest validator was complaining on this field missing.  I can see other fields like Description where we have a "[Outlook Add-in description]" default value and Provider with "[Provider Name]".  So we can either have a place holder, e.g. "[Support URL]" or not fail the manifest validation when this field is not supplied or supplied but empty.  Thoughts?

P.S. Thanks for reviewing the PR.